### DISCUSSION TITLE
Rename (or delete and make new) metric for a run

### DISCUSSION THREAD

**Discussion by dr-costas** [mlflow/mlflow#7690] [Q&A] [answered]
Hi, 

I have some finished runs on an experiment and I decided that I want to change the naming of some metrics. How can I do this? 

Can I, for example, rename the existing metrics somehow? Or, can I delete the existing metrics and then log the new ones?

**Comment by JHibbard** [comment, marked as answer]
Hi dr-costas,

After looking through the REST API, there doesn't appear to be a way to rename or remove a metric once it has been logged. Depending on the complexity of your logging setup, you could version your metric naming scheme in an external document and tag experiments based on which naming scheme they used. This may help you query across the two metric names.

Hope that helps!
Jim
